cfgov/unprocessed/apps/tccp/js/index.js

```diff
@@ -27,7 +28,40 @@ function init() {
   // Initialize any tooltips on the page
   initializeTooltips();
   // Reinitialize tooltips after an htmx request replaces DOM nodes
-  attach(document, 'htmx:afterSwap', initializeTooltips);
+  attach(document, 'htmx:afterSwap', initializeAndReport);
+}
+
+/**
+ *
+ * @param {Event} event - htmx event
+ */
+function initializeAndReport(event) {
+  initializeTooltips();
+  reportFilter(event);
+}
+
+/**
+ *
+ * @param {Event} event - htmx event
+ */
+function reportFilter(event) {
+  let category, value;
+  const { target } = event.detail.requestConfig.triggeringEvent;
+  if (target.tagName === 'SELECT') {
+    category = target.name;
+    value = target.value;
+  } else {
+    category = target
+      .closest('fieldset')
+      .firstElementChild.textContent.toLowerCase();
+    value = target.labels[0].textContent.trim();
+  }
+
+  analyticsSendEvent({
+    name: 'tccp:card-list-refinement',
```

Comment:
The analytics team had just one requested change:

```suggestion
    event: 'tccp:card-list-refinement',
```